lib/api.js

```diff
@@ -62,7 +62,12 @@ async function processDocs(driver, project) {
 
     await driver.write(
       'versions',
-      [{ id: project, name: project, versions }],
+      [{ 
+        id: project, 
+        name: project,
+        index_date_timestamp: Date.now(), 
```

Comment:
Could we run into problems if people from different regions ran the script?

Reply:
I don't think so? It's Heroku Scheduler that runs the script at an interval, so the timezone would be what Heroku uses.

Reply:
I think @ijlee2 is correct, the Heroku Scheduler should be the only thing executing this script, but yes if we ran the script locally I suppose we could run into issues. 

This approach is meant as a temporary fix until we can better assess how much of the process could be refactored or improved. 

FWIW I did confirm via some manual testing that this logic finds the correct index object.